set.mm

```diff
@@ -237805,6 +238079,472 @@ factorization into prime power factors (even if the exponents are
       ACDIUAJKLMNUUBUUC $.
   $}
 
+$(
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  Simple groups
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+$)
+
+  $c SimpGrp $.
+
+  $( Extend class notation with the class of simple groups. $)
+  csimpg $a class SimpGrp $.
+
+  $( Define class of all simple groups.  A simple group is a group ( ~ df-grp )
+     with exactly two normal subgroups.  These are always the subgroup of all
+     elements and the subgroup containing only the identity ( ~ simpgnsgbid ).
+     (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+  df-simpg $a |- SimpGrp = { g e. Grp | ( NrmSGrp ` g ) ~~ 2o } $.
+
+  ${
+    $d g G $.
+    $( The predicate "is a simple group".  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
+    issimpg $p |- ( G e. SimpGrp <-> ( G e. Grp /\ ( NrmSGrp ` G ) ~~ 2o ) ) $=
+      ( vg cnsg cfv c2o cen wbr cgrp csimpg wceq fveq2 breq1d df-simpg elrab2
+      cv ) BOZCDZEFGACDZEFGBAHIPAJQREFPACKLBMN $.
+  $}
+
+  ${
+    issimpgd.1 $e |- ( ph -> G e. Grp ) $.
+    issimpgd.2 $e |- ( ph -> ( NrmSGrp ` G ) ~~ 2o ) $.
+    $( Deduce a simple group from its properties.  (Contributed by Rohan
+       Ridenour, 3-Aug-2023.) $)
+    issimpgd $p |- ( ph -> G e. SimpGrp ) $=
+      ( cgrp wcel cnsg cfv c2o cen wbr csimpg issimpg sylanbrc ) ABEFBGHIJKBLFC
+      DBMN $.
+  $}
+
+  $( A simple group is a group.  (Contributed by Rohan Ridenour,
+     3-Aug-2023.) $)
+  simpggrp $p |- ( G e. SimpGrp -> G e. Grp ) $=
+    ( csimpg wcel cgrp cnsg cfv c2o cen wbr issimpg simplbi ) ABCADCAEFGHIAJK
+    $.
+
+  ${
+    simpggrpd.1 $e |- ( ph -> G e. SimpGrp ) $.
+    $( A simple group is a group.  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
+    simpggrpd $p |- ( ph -> G e. Grp ) $=
+      ( csimpg wcel cgrp simpggrp syl ) ABDEBFECBGH $.
+  $}
+
+  $( A simple group has two normal subgroups.  (Contributed by Rohan Ridenour,
+     3-Aug-2023.) $)
+  simpg2nsg $p |- ( G e. SimpGrp -> ( NrmSGrp ` G ) ~~ 2o ) $=
+    ( csimpg wcel cgrp cnsg cfv c2o cen wbr issimpg simprbi ) ABCADCAEFGHIAJK
+    $.
+
+  ${
+    trivnsimpgd.1 $e |- B = ( Base ` G ) $.
+    trivnsimpgd.2 $e |- .0. = ( 0g ` G ) $.
+    trivnsimpgd.3 $e |- ( ph -> G e. Grp ) $.
+    trivnsimpgd.4 $e |- ( ph -> B = { .0. } ) $.
+    $( Trivial groups are not simple.  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
```

Comment:
Why? Because they have one and only one normal subgroup (because the subgroup of all elements **is** the subgroup containing only the identity). Maybe this can be added to the comment.

Reply:
I think this trivial (sic) remark can be avoided :-)

Reply:
OK